app/models/project.rb

```diff
@@ -114,6 +114,13 @@ class Project < ActiveRecord::Base
   before_destroy :delete_all_members
   before_destroy :destroy_all_work_packages
 
+  # HACK: remove on upgrade to awesome_nested_set 3.
+  # Active Record 4 will, by default, order by id:
+  #   ORDER BY "projects"."id" ASC LIMIT 1
+  # this results in the following genre of errors on PostgreSQL:
+  #   …it is not contained in either an aggregate function or the GROUP BY clause
+  default_scope order('')
```

Comment:
@ulferts specifying that the `default_scope` orders by name might work here though (and make for a better default).